CLA.md

```diff
@@ -85,7 +88,7 @@ backup e-mail support address <login-issues@jira.linuxfoundation.org>
 ## Setting up the CNCF CLA check
 
 If you are a Kubernetes GitHub organization or repo owner and would like to setup
-the Linux Foundation CNCF CLA check for your repositories, [read the docs on setting up the CNCF CLA check](/github-management/setting-up-cla-check.md)
+the Linux Foundation CNCF CLA check for your repositories, kindly read the [docs on setting up the CNCF CLA check](/github-management/setting-up-cla-check.md)
```

Comment:
We prefer the original in this case. For docs we prefer to not add modifiers. For an explanation see the style guide on simple and direct language:

https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language

Reply:
I have undone the changes. Thanks for letting me know!